src/main/resources/templates/login.html

```diff
@@ -1,68 +1,68 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
 
-<head th:replace="~{fragments/head-fragments :: head('User profile page')}"><title></title></head>
+<head th:replace="~{fragments/head-fragments :: head('Login')}"><title></title></head>
 
 <body>
 <section class="container-login-page">
 
-    <div class="container-login">
+    <!--/*@thymesVar id="registerStatusMsg" type=""*/-->
+    <h1 th:if="${registerStatusMsg != null}"
+        th:text="${registerStatusMsg}"
+        class="position-absolute top-0 start-0 text-success"
+    >
+    </h1>
+    <form th:action="@{/auth/login}"
+          method="post"
+          th:method="post"
+    >
+        <h3 class="text-white text-center">Please login</h3>
 
-        <h1 th:if="${registerStatusMsg != null}"
-            th:text="${registerStatusMsg}"
-            style="position: absolute; bottom: 50px; left: 50%; transform: translateX(-50%); color: darkgreen"
-        >
-        </h1>
-        <form th:action="@{/auth/login}"
-              method="post"
-              th:method="post"
-        >
-            <h2 class="text-white text-center">Welcome back</h2>
-            <!-- Email input -->
-            <div class="form-outline mb-4">
-                <input type="text"
-                       id="usernameOrEmail"
-                       th:name="usernameOrEmail"
-                       class="form-control"/>
-                <label class="form-label" for="usernameOrEmail">Username or email address</label>
-            </div>
-
-            <!-- Password input -->
-            <div class="form-outline mb-4">
-                <input type="password"
-                       id="password"
-                       th:name="password"
-                       class="form-control"/>
-                <label class="form-label" for="password">Password</label>
-            </div>
+        <hr>
+        <!-- Email input -->
+        <div class="form-floating ">
```

Comment:
"form-floating text-muted"

Reply:
text-mutted is a css class which has the color code: #6c757d 
But in our login.css we defined a custom color code for our form which is color: #ccc6c0; 
Refference: login.css line 13